aten/src/ATen/native/native_functions.yaml

```diff
@@ -1215,6 +1215,8 @@
 - func: mvlgamma_(Tensor self, int64_t p) -> Tensor
   variants: method
 
+- func: narrow_copy(Tensor self, int64_t dim, int64_t start, int64_t length) -> Tensor
```

Comment:
On master, the native functions by default only generate function variant, i.e., `at::xxxx`. So you can't use `tensor.narrow_copy`. You probably want this method-only, in which case you can do `variants: method`. You can rebase onto master to reproduce the build failures.